api/web3server_utils.go

```diff
@@ -412,6 +394,13 @@ func hexToBytes(str string) ([]byte, error) {
 	return hex.DecodeString(str)
 }
 
+func getLogsBloomFromBlkMeta(blkMeta *iotextypes.BlockMeta) string {
+	if len(blkMeta.LogsBloom) == 0 {
+		return "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
1. this should also have "0x" prefix?
2. define a const _zeroBloom = "0x000..."